### PR TITLE
Completed Typescript Forms Exercise-Allyson Greven

### PR DESCRIPTION
**Pull request by allygreven**
Screenshot 2024-11-07 at 11 56 57 AM (2)

### CODE REVIEW

File: typescript-forms/form-events.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  event.preventDefault();
+  const $formElements = $contactForm.elements as FormElements;
+  const formData = {
+    name: $formElements.name.value,
+    email: $formElements.email.value,
+    message: $formElements.message.value,
+  };
+  console.log(formData);
+  $contactForm.reset();
+});
```
Comment (shawnkost) [inline typescript-forms/form-events.ts:59]: This is in the wrong file

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Ally

File: typescript-forms/form-events.ts (hunk 1 above)
Comment (allygreven) [reply typescript-forms/form-events.ts:59]: oh shoot okay i'll change, thanks!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**